src/main/scala/de/heikoseeberger/sbtheader/HeaderPlugin.scala

```diff
@@ -140,31 +146,34 @@ object HeaderPlugin extends AutoPlugin {
             .value
             .toList ++ unmanagedResources.in(headerCreate).value.toList
         ),
-        headers.value,
+        headerLicense.value,
+        headerMappings.value,
         streams.value.log
       )
     )
 
   private def notToBeScopedSettings =
     Vector(
-      headers := Map.empty,
+      headerMappings := Map.empty,
       headerExcludes := Seq.empty
```

Comment:
I don't think so. I've investigated a bit in that direction (see #76) but I wanted to have a full sat-scripted test that shows that we don't need it.